sdk/lib/developer/extension.dart

```diff
@@ -146,13 +146,38 @@ external bool get extensionStreamHasListener;
 /// event stream.
 ///
 /// If [extensionStreamHasListener] is false, this method is a no-op.
-void postEvent(String eventKind, Map eventData) {
+// This event will only be posted to the extension stream, if you try to post to a stream that is part of the vm service or is defined by the dart VM Embedder
```

Comment:
Woahhhhh Nelly, slow down! 80 character lines in the SDK ;)

Reply:
This documentation needs to be reworded and should explicitly document the behavior of `stream`.